lib/web_ui/lib/src/engine/surface/platform_view.dart

```diff
@@ -57,16 +57,16 @@ class PersistedPlatformView extends PersistedLeafSurface {
   }
 
   @override
-  Matrix4 get localTransformInverse => null;
+  Matrix4? get localTransformInverse => null;
 
   @override
   void apply() {
-    rootElement.style
+    rootElement!.style
```

Comment:
In the `PersistedSurface` this is defined as:

```dart
  html.Element? rootElement;
```

How can you be sure here that rootElement is not going to be null? (I mean, it's nullable after all?)

Reply:
(Asking because of the note in the null safety [docs](https://dart.dev/null-safety#using-variables-and-expressions):

>  **Important:** If you aren’t positive that a value is non-null, **don’t use !**.

, is this just to _"let me write my code as if non-null, but if null fail at runtime like before null safety"_?)

Reply:
The same way we knew it before :) The new code preserves the old semantics. However, in follow-up PRs we could clean this up. Ideally, we wouldn't use `!` for null assertions at all, but it will take time for us to get there.

Reply:
Preserving the old semantics makes sense, thanks!!